environments/kafkaconnect/base/kafka-connect-strimzi.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnect
+metadata:
+  name: connect-cluster-<UNIQUE_ID>
+  namespace: vaccine-omo
```

Comment:
We probably want this `namespace` reference deleted.

Reply:
addressed in the latest commits